packages/help-center/src/hooks/use-active-support-tickets-query.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-restricted-imports */
```

Comment:
This was extracted [from here](https://github.com/Automattic/wp-calypso/blob/trunk/client/data/help/use-active-support-tickets-query.js)
I was undecided on whether or not to move it to `@automattic/data-stores`, and it ended up here.
Any strong opinions?

Reply:
Not really. We can move it if we decide we need to.